tests/integration_tests/build/test_licenses.py

```diff
@@ -0,0 +1,43 @@
+
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests ensuring codebase contains neccessary licenses."""
+
+import os
+
+import pytest
+
+CHROMIUM_COPYRIGHT = "Copyright 2017 The Chromium OS Authors. All rights reserved."
+CHROMIUM_LICENSE = "Use of this source code is governed by a BSD-style license that can be"
+AMAZON_COPYRIGHT = "Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved."
+AMAZON_LICENSE = "SPDX-License-Identifier: Apache-2.0"
+EXCLUDED_DIRECTORIES = ((os.path.join(os.getcwd(), 'build')))
+
+# Python and Rust files should have the licenses on the first 2 lines.
```

Comment:
This is actually documentation, so you should include it in the documentation string after the `def validate_license(filename)` line.